app/src/main/java/com/ncnf/views/fragments/organization/OrganizationTabFragment.java

```diff
@@ -190,7 +190,7 @@ private boolean verifyTextInput(EditText input) {
         return true;
     }
 
-    private void displayPopUp(View view, String errorText) {
+    private void displayPopUp(String errorText) {
```

Comment:
thanks but this will be changed

Reply:
I removed the view argument as it wasn't used in the method